### PR TITLE
Feature(core/ts-version): add orderbook controller and refactor order…

### PR DESCRIPTION
**Pull request by WilliamHawkingStephen**
…book output type and add getAll orders functionality to order repository

## Check list  

- [x] All Merge Request pipelines should be passed.
- [x] Make sure all the changes are necessary and correct.
- [x] Make sure no needed change missing (all necessary changes are made).
- [x] Merge Request should contains only changes for one purpose (one feature or bug or refactoring).
- [ ] All necessary unit tests are written.
- [x] Critical changes should be tested locally (or on staging).

### CODE REVIEW

File: ts-version/src/presentation/http/express.ts [hunk 1]
```diff
@@ -82,11 +84,11 @@ export default class HttpServer {
     this.expressApp.delete("/user/order", auth, (req, res, next) =>
       this.orderController.cancel(req, res, next).catch(next),
     );
+    this.expressApp.get("/orderbook", this.orderBookController.getOrderBook);
```
Comment (franken5tein) [inline ts-version/src/presentation/http/express.ts:87]: In order to have request and response, the request and response must be given to the controller as input

File: ts-version/src/presentation/http/controllers/orderbook.controller.ts [hunk 2]
```diff
@@ -3,23 +3,23 @@ import { IOrderbookService } from "@port/orderbook-service.interface";
...
-      side: side as unknown as OrderSide,
+      network: network as string,
+      market: market as string,
+      side: side as OrderSide,
       page: page as unknown as number,
-      perPage: per_page as unknown as number,
+      perPage: perPage as unknown as number,
     });
 
     res.send(response);
```
Comment (franken5tein) [inline ts-version/src/presentation/http/controllers/orderbook.controller.ts:22]: If the service did not respond, a log should have been sent

File: ts-version/src/domain/services/orderbook/orderbook.ts [hunk 3]
```diff
@@ -4,13 +4,17 @@ import { IOrderbookService } from "@port/orderbook-service.interface";
...
+import { IDataService } from "@port/data-service.interface";
+import { SupportedNetworks } from "@type/supportedNetwrok";
 export default class OrderBookService
   extends Observerable
   implements IOrderbookService.schema {
 
-  constructor(private OrderAdapter: OrderRepoAdapter) {
+  constructor(
+    private OrderAdapter: OrderRepoAdapter,
+    private dataService: IDataService.schema,
```
Comment (franken5tein) [inline ts-version/src/domain/services/orderbook/orderbook.ts:16]: It must not be possible for a service to be used by another service

File: ts-version/src/domain/services/orderbook/orderbook.ts [hunk 4]
```diff
@@ -4,13 +4,17 @@ import { IOrderbookService } from "@port/orderbook-service.interface";
 import Utilities from "src/domain/Utilities";
 import OrderRepoAdapter from "src/data/database/postgres/repository/order.adapter";
 import Redis from "src/data/database/redis";
```
[the remark this answers is not in the document]
Comment (WilliamHawkingStephen) [reply ts-version/src/domain/services/orderbook/orderbook.ts]: done

**Pull request closed without merging**